test/RetireNet.Runtimes.Core.Tests/RetireNet.Runtimes.Core.Tests.csproj

```diff
@@ -1,14 +1,15 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
         <TargetFramework>netcoreapp2.1</TargetFramework>
         <RootNamespace>RetireRuntimeMiddleware.Tests</RootNamespace>
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.0.1" />
-        <PackageReference Include="xunit" Version="2.4.0" />
-        <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
+        <PackageReference Include="coverlet.msbuild" Version="2.7.0" PrivateAssets="All" />
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.3.0" />
+        <PackageReference Include="xunit" Version="2.4.1" />
+        <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1" PrivateAssets="All" />
```

Comment:
Just curious: since this a test project, why do you specify the `PrivateAssets=All`?

Reply:
TBH this came automatically with the regular NuGet update. I guess this happens to every package, which is designed to be not a dependency in referencing projects. In this case, it doesn't make sense, but imagine you want to use a static analyzer package such as `Microsoft.CodeAnalysis.FxCopAnalyzers` in your Tool-project, then you typically don't want warnings and errors from it in your referencing test projects and you also don't want the `.dll`  in your own NuGet package build.

You will find detailed information here: https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#controlling-dependency-assets

Reply:
Ah, right :) No biggie, though - just a question on my end.